minidump/src/minidump.rs

```diff
@@ -1649,6 +1722,404 @@ impl<'a> MinidumpMemoryInfo<'a> {
     }
 }
 
+impl<'a> MinidumpStream<'a> for MinidumpLinuxMaps<'a> {
+    const STREAM_TYPE: MINIDUMP_STREAM_TYPE = MINIDUMP_STREAM_TYPE::LinuxMaps;
+
+    fn read(
+        bytes: &'a [u8],
+        _all: &'a [u8],
+        _endian: scroll::Endian,
+    ) -> Result<MinidumpLinuxMaps<'a>, Error> {
+        let regions = str::from_utf8(bytes)
```

Comment:
Per above I'm concerned about this making data silently unavailable if non-well-formed-utf-8 filenames are present in the contents. I know it's slightly more annoying, but using `bytes..split(|b| b == b' ')` here, then tweaking `from_line` to parse from bytes by splitting on spaces and stringifying everything but the filename would avoid that.

That being said, given that it's an edge case I fully support you filing an issue for someone to fix later, noting it here, and moving on in life.

Reply:
Ugh yeah you're right. Gross.

Reply:
You've done enough here, let's get this merged and we can deal with the fiddly shit later. It shouldn't impact the public API much, it's just implementation details.